configs/syscfg.txt

```diff
@@ -0,0 +1,9 @@
+# This file is intended to contain system-made configuration changes. User
+# configuration changes should be placed in "usercfg.txt". Please refer to the
+# README file for a description of the various configuration files on the boot
+# partition.
+
+dtparam=i2c_arm=on
+dtparam=spi=on
+
+include nobtcfg.txt
```

Comment:
Ah, so this is the place which the bluetooth package will modify to switch bluetooth on and off, right?

Reply:
This is indeed the intention; flip this to `btcfg.txt` and back to switch between the configs (and eventually we'll have a CLI tool to handle this and the manipulation of the dtparam lines above).